pkgs/development/php-packages/grumphp/default.nix

```diff
@@ -1,11 +1,11 @@
 { mkDerivation, fetchurl, makeWrapper, lib, php }:
 
-mkDerivation rec {
+mkDerivation (finalAttrs: {
```

Comment:
So, if I understand correctly, we have to do that for all the packages in `pkgs/development/php-packages/` ?

Reply:
No, its just to prove https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes works in packages using this mkDerivation.